eng/Tools.props

```diff
@@ -52,6 +52,35 @@
     <RepoTool Include="dotnet-reportgenerator-globaltool" Version="$(DotNetReportGeneratorGlobalToolPackageVersion)" />
   </ItemGroup>
 
+  <Target Name="InitOptionalTools"
```

Comment:
Is this a common pattern to put a `Target` in a `.props` file?

Reply:
For tool bootstrapping yes as it is invoked by the arcade toolset.